.github/scripts/github_utils.py

```diff
@@ -179,3 +179,24 @@ def gh_fetch_merge_base(org: str, repo: str, base: str, head: str) -> str:
 def gh_update_pr_state(org: str, repo: str, pr_num: int, state: str = "open") -> None:
     url = f"{GITHUB_API_URL}/repos/{org}/{repo}/pulls/{pr_num}"
     gh_fetch_url(url, method="PATCH", data={"state": state})
+
+
+def get_pr_reviews(org: str, repo: str, pr_num: int) -> List[Dict[str, Any]]:
```

Comment:
Can you improve the type hints to better explain what this return value contains?

And actually, is this method even used?

Reply:
I think it was an artifact of git messing up a merge.